contrib/tools/workflowcheck/determinism/ident_refs.go

```diff
@@ -38,6 +38,10 @@ var DefaultIdentRefs = IdentRefs{
 	// We mark these as deterministic since they give so many false positives
 	"(reflect.Value).Interface": false,
 	"runtime.Caller":            false,
+	"fmt.Errorf":                false,
+	"fmt.Sprintf":               false,
+	"fmt.Sprint":                false,
+	"fmt.Sprintln":              false,
```

Comment:
May want to add each of these being called in a method in `contrib/tools/workflowcheck/determinism/testdata/src/a/calls.go` but not that big of a deal.

Also, may want to add the `Append` and `Sscan` funcs here too

Reply:
Yeah, just for peace of mind, can we add the following to the bottom of `contrib/tools/workflowcheck/determinism/testdata/src/a/calls.go`:

```go
func SafeFmtCall() {
	fmt.Sprintf("foo bar")
}
```

Reply:
sure